scripts/install

```diff
@@ -70,6 +70,20 @@ ynh_setup_source "$final_path"
 # NGINX CONFIGURATION
 #=================================================
 
+# Do not serve .well-known if it's already served on the domain
+if https://github.com/YunoHost-Apps/nextcloud_ynh/blob/962cc61ec8ffb045d89dd460cf8b55778d862ffc/scripts/install "https://${domain}/.well-known/caldav" ; then
```

Comment:
I guess you would write `is_url_handled` ?

Reply:
Well yes... copy pasta